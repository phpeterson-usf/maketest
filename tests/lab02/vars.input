$(TESTS_DIR)/vars.ntc
