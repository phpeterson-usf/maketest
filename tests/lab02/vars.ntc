x = 2
y = x + 100
