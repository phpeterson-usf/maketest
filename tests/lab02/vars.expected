TK_IDENT("x")
TK_EQ("=")
TK_INTLIT("2")
TK_EOL("")
TK_IDENT("y")
TK_EQ("=")
TK_IDENT("x")
TK_PLUS("+")
TK_INTLIT("100")
TK_EOL("")
TK_EOT("")
